locales-pending/premium.ftl

```diff
@@ -53,6 +53,29 @@ modal-cta-ok = OK
 modal-open-alt = Open modal
 modal-close-alt = Close modal
 
+# About Exposure Types Modal
+
+modal-exposure-type-title = About exposure types
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+modal-exposure-type-description = We search for you in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Here are the two types of exposures we find:
+modal-exposure-type-data-breach = <b>Data breach</b> means your information has been compromised in a breach and could be in the wrong hands. 
+  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-premium }.
+modal-exposure-type-data-broker = <b>Info for sale</b> means a data broker site is publicly publishing and selling your personal info. 
+  You’ll need to manually request removal from each site. But if you’re a { -brand-premium } user, we auto-removal all profiles for you. 
+  In both cases, Removals typically take 7-14 days. Some can take longer, while others can happen within the hour.
+
+# About Exposure Types Modal
+
+modal-exposure-status-title = About exposure statuses
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+modal-exposure-status-description = We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info.
```

Comment:
Make this a plural string anyway.

Also: "it will always be plural" -> "it will always be more than 1".